src/groups/group_action.jl

```diff
@@ -112,6 +123,29 @@ function apply_diff!(A::AbstractGroupAction, Y, a, p, X)
     )
 end
 
+@doc raw"""
+    apply_diff_group(A::AbstractGroupAction, a, X, p)
+
+Compute the value of differential of action [`AbstractGroupAction`](@ref) `A` on vector `X`,
+where element `a` is acting on `p`, with respect to the group element.
+
+Let ``\mathcal G`` be the group acting on manifold ``\mathcal M`` by the action `A`.
+The action is of element ``g ∈ \mathcal G`` on a point ``p ∈ \mathcal M``.
+The differential transforms vector `X` from the tangent space at `a ∈ \mathcal G`, 
+``X ∈ T_a \mathcal G`` into the tangent space of manifold ``\mathcal M``.
```

Comment:
Now its just that X is double in this line and the one before? The rest is fine :) - and which tangent space at M? The one at p?

Reply:
The result is from `T_{τ_a p} \mathcal M`, as defined below. Feel free to suggest an edit if you have a good idea :slightly_smiling_face: .

Reply:
ah I missed that the tangent space is given below, so we could just write _a_ tangent space (instead of the) here?